spec/models/ability_spec.rb

```diff
@@ -113,23 +120,38 @@
     it { is_expected.to be_able_to(:view_content, dro) }
   end
 
-  context 'with the edit role' do
-    let(:roles) { ['dor-apo-metadata'] }
+  context 'with the manage role on the cocina_object' do
+    let(:new_cocina_object_roles) { ['dor-apo-manager'] }
+
+    it { is_expected.not_to be_able_to(:manage, :everything) }
+    it { is_expected.not_to be_able_to(:manage_item, dro) }
+    it { is_expected.not_to be_able_to(:manage_governing_apo, dro, apo_id) }
+    it { is_expected.not_to be_able_to(:manage_desc_metadata, dro) }
+    it { is_expected.not_to be_able_to(:create, Cocina::Models::AdminPolicy) }
+
+    it { is_expected.not_to be_able_to(:view_metadata, dro) }
+    it { is_expected.not_to be_able_to(:view_metadata, collection) }
+    it { is_expected.to be_able_to(:view_metadata, admin_policy) }
+    it { is_expected.not_to be_able_to(:view_content, dro) }
```

Comment:
This shows the restored ability (line 134) and also that no other new abilities are conferred with the change above.